.github/workflows/ci.yml

```diff
@@ -180,7 +180,10 @@ jobs:
                   git submodule update --init \
                   ;'
             - name: Rustfmt check
-              run: cargo fmt -- --check
+              uses: actions-rs/cargo@v1
+              with:
+                  command: fmt
+                  args: -- --check
```

Comment:
I believe the highlighting offered by cargo action (compared to raw cargo call) shows like this in practice:

![image](https://user-images.githubusercontent.com/315866/183277410-14feb4e0-59ad-4cd5-94fd-736fe3414016.png)
![image](https://user-images.githubusercontent.com/315866/183277429-8ee9cceb-5a48-4c57-a650-c39c0997a0d2.png)